scripts/create_person.rb

```diff
@@ -0,0 +1,10 @@
+caren = Licensee.create surname: 'Bishop',
+                      given_name: 'Caren',
+                      gender: 'f',
+                      height: 63,
+                      weight: 132,
+                      born_on: '1943-09-26',
+                      eye_color: 'Brown',
+                      hair_color: 'Black'
+
+p caren
```

Comment:
Also, while it contradicts the directions, I like your instinct of keeping scripts separate and putting them in a separate directory.